src/components/FeatureCard/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { StaticImageData } from "next/image";
+import React from "react";
+
+type cardProps = {
+  header: string;
+  description: string;
+  src: StaticImageData;
+  alt: string;
+};
+
+function FeatureCard({ header, description, src, alt }: cardProps) {
+  return (
+    <div className="flex justify-between space-x-48 items-center mt-32">
+      <div>
```

Comment:
As a general rule, I try avoid adding style positioning inside the component definition (I'm referring to the margin top 32). Rather I put it on the parent component via flex gap, margin or padding or or some other positioning style. In that way we make the FeatureCard more flexible and could potentially be used in different situations